api/app/src/routes/user.ts

```diff
@@ -40,6 +44,40 @@ router.get(
   })
 );
 
+/**
+ * GET /user/user
+ *
+ * Gets all users and their connected providers for the specified customer.
+ *
+ * @return { ConnectedUserInfo[] }
+ */
+router.get(
+  "/user",
```

Comment:
to be explicit, we don't want `/user/user` -> just `/user`

Reply:
Fixed!